docs/platform_service_status.md

```diff
@@ -0,0 +1,27 @@
+# Platform Service Status
+
+This provides a status of our overall services and where in the lifecycle they are in their evolution.
+Service lifecycle definitions can be found [here](../docs/service_lifecycles.md). This is to help projects understand where the gaps might be on some of the services they might want to consume and therefore assess the risk related to them.
```

Comment:
"This provides the status of all of our services in regards to where they are in the development lifecycle" maybe nicer